src/ModCache.ts

```diff
@@ -56,21 +56,16 @@ export class ModCache {
     }
 
     /**
-     * Populate the cache by fetching moderation room membership events
+     * Populate the cache by fetching moderation room members
      */
     public async populateCache() {
-        const memberEvents = await this.client.getRoomMembers(
-            this.managementRoomId,
-            undefined,
-            ["join", "invite"],
-            ["ban", "leave"],
-        );
+        const members = await this.client.getJoinedRoomMembers(this.managementRoomId);
         this.modRoomMembers = [];
-        memberEvents.forEach((event) => {
-            if (!this.modRoomMembers.includes(event.stateKey)) {
-                this.modRoomMembers.push(event.stateKey);
+        members.forEach((member) => {
+            if (!this.modRoomMembers.includes(member)) {
+                this.modRoomMembers.push(member);
             }
```

Comment:
it occurs to me that we could use a `Set` here to avoid this kind of logic. This isn't necessarily something for this PR to fix